data/data/aws/vpc/sg-worker.tf

```diff
@@ -146,6 +146,26 @@ resource "aws_security_group_rule" "worker_ingress_kubelet_insecure_from_master"
   to_port   = 10250
 }
 
+resource "aws_security_group_rule" "worker_ingress_internal" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.worker.id}"
+
+  protocol  = "tcp"
+  from_port = 9000
+  to_port   = 9999
```

Comment:
This should sort in before the insecure kubelet rule (at 10250) and replace the existing 9100 rule.  See #683 for an example).

Reply:
done hopefully

Reply:
> done hopefully

Yup.  Thanks :)